packages/emulators/standalone/ppsspp-sa/sources/RK3566/ppsspp.ini

```diff
@@ -509,3 +535,23 @@ ColorCorrectionSettingValue4 = 1.000000
 ScanlinesSettingValue1 = 1.000000
 ScanlinesSettingValue2 = 0.500000
 SharpenSettingValue1 = 1.500000
+[VR]
```

Comment:
Do we require any VR related config?

Reply:
Hi @edemirkan ,

We don't, this shouldn't be relevant for Linux at all, PPSSPP under Android supports VR. My process for creating this .ini was:

1. Launch PPSSPP on Rocknix
2. Apply and test settings
3. Upload .ini

The VR section seems to be added automatically by PPSSPP if it's not in the file and it seems that some PPSSPP.ini files were added before this functionality was introduced (see e.g RK3266, this already includes the [VR] section, RK3566 did not).

If you would prefer, I'm happy to rework this PR to minimize the diff but I figured it would be a good opportunity to minimize the changes between the different systems instead.